ArticleTemplates/assets/js/modules/creativeInjector.js

```diff
@@ -13,35 +13,60 @@ function (
         if (!initialised) {
             initialised = true;
 
-            window.injectCreative = injectCreative;
-            window.applyNativeFunctionCall('injectInlineCreative');
+            if (GU.opts.contentType === 'liveblog' || document.querySelector('.article__body--liveblog')) {
+                trackLiveBlogEpic();
+            } else {
+                window.injectCreative = injectCreative;
+                window.applyNativeFunctionCall('injectInlineCreative');
+            }
+        }
+    }
+
+    function trackLiveBlogEpic() {
+        var liveBlogEpicContainer = document.querySelector('contributions-epic');
```

Comment:
I must've missed this before but the querySelector needs a '.' prepended to it to select by class name. eg '.contributions-epic'.

You could also update the selector to be '.contributions-epic:not([data-tracked])' which will filter out all those elements already tracked. By doing this we don't need the if block on line 32.

Now that we can potentially have more than 1 epic in a liveblog we should also use querySelectorAll as querySelector only returns the first matching element.